composer.json

```diff
@@ -12,7 +12,18 @@
         "php": ">=7.1",
         "easyrdf/easyrdf": "0.10.0-alpha.1 as 0.9.2"
     },
-    "require-dev": {
-        "minimaxir/big-list-of-naughty-strings": "dev-master"
-    }
+    "repositories": [
```

Comment:
Now that the library with strings as been moved, we can drop also the `repositories` entry. Also, it should be removed from Joinup as the test is here now.